### PR TITLE
Fix to wait until all docs loaded in quick start

### PR DESCRIPTION
**Pull request by jackjlli**
This PR fixes to wait until all docs loaded in quick start.

### CODE REVIEW

**Comment by jackjlli** [comment]
@kishoreg The code is in `pinot-tool` module, and we don't have any test in `pinot-tool` though.

**Comment by kishoreg** [comment]
this is not the right fix. It just masking NPE. The real problem is in the following lines
```    
printStatus(Color.CYAN, "***** Waiting for 5 seconds for the server to fetch the assigned segment *****");   
Thread.sleep(5000);
```
Similar to what we have in integration tests, we need to wait until the segments are loaded and select count(*) returns the right number.

**Comment by jackjlli** [comment]
@kishoreg Code updated

File: pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java [hunk 1]
```diff
@@ -209,6 +212,35 @@ public void run() {
     printStatus(Color.GREEN, "You can always go to http://localhost:9000/query/ to play around in the query console");
   }
 
+  protected void waitForAllDocsLoaded(QuickstartRunner runner, long timeoutMs)
+      throws Exception {
+    final long countStarResult = 97889L;
+    TestUtils.waitForCondition(new Function<Void, Boolean>() {
```
Comment (kishoreg) [inline pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java]: TestUtils? better to use RetryPolicies in PinotCommon. It allows you to try every 5 seconds until 60 seconds.

File: pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java [hunk 2]
```diff
@@ -158,8 +161,8 @@ public void execute()
     runner.buildSegment();
     printStatus(Color.CYAN, "***** Pushing segment to the controller *****");
     runner.pushSegment();
-    printStatus(Color.CYAN, "***** Waiting for 5 seconds for the server to fetch the assigned segment *****");
-    Thread.sleep(5000);
+    printStatus(Color.CYAN, "***** Waiting for 60 seconds for the server to fetch the assigned segment *****");
+    waitForAllDocsLoaded(runner, 6000L);
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java]: Done.

File: pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java (hunk 1 above)
Comment (jackjlli) [reply pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java]: Changed to `RetryPolicies`.

**Comment by npawar** [comment]
Closed this PR due to 6 months inactivity. Reopen if needed.

**Pull request closed without merging**